src/test/resources/jenkins/plugins/git/GitSCMSourceTraitsTest/cleancheckout_v1_trait.xml

```diff
@@ -0,0 +1,11 @@
+<jenkins.plugins.git.GitSCMSource>
+  <traits>
+    <jenkins.plugins.git.traits.BranchDiscoveryTrait/>
```

Comment:
I guess the `BranchDiscoveryTrait` is left from an export. I would remove it as the tests are only about `Clean` traits as you also commented in the tests.

Reply:
I'd need to do conditional logic in tests because `BranchDiscoveryTrait` is added implicitly when loading `<extensions>` but not when loading `<traits>`.

Reply:
Ok. Got it.

Reply:
Ok, done.